observability/micrometer/src/main/resources/org/wildfly/extension/micrometer/LocalDescriptions.properties

```diff
@@ -5,3 +5,5 @@ micrometer.remove=Operation Removes subsystem
 micrometer.exposed-subsystems=The names of the WildFly subsystems that exposes their metrics (or '*' to expose any subsystem metrics).
 micrometer.prefix=Prefix prepended to the name of the WildFly metrics exposed by the HTTP endpoints.
 micrometer.security-enabled=True if authentication is required to access the HTTP endpoint on the HTTP management interface.
+micrometer.endpoint=The address to where metrics will be published. The OTLP default is http://localhost:4318/v1/metrics, though WildFly provides no default, so this must be configured manually.
```

Comment:
This is wrong. When I boot the server without any manual configuration I am getting failures to connect.

```
[standalone@embedded /] 17:46:00,011 WARNING [io.micrometer.registry.otlp.OtlpMeterRegistry] (otlp-metrics-publisher) Failed to publish metrics to OTLP receiver: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:412)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:255)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:237)
	at java.base/java.net.Socket.connect(Socket.java:609)
	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:177)
	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:507)
	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:602)
	at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:275)
	at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:374)
	at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:395)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1253)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1187)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1081)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1015)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getOutputStream0(HttpURLConnection.java:1367)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getOutputStream(HttpURLConnection.java:1342)
	at io.micrometer@1.9.3//io.micrometer.core.ipc.http.HttpUrlConnectionSender.send(HttpUrlConnectionSender.java:98)
	at io.micrometer@1.9.3//io.micrometer.core.ipc.http.HttpSender$Request$Builder.send(HttpSender.java:305)
	at io.micrometer@1.9.3//io.micrometer.registry.otlp.OtlpMeterRegistry.publish(OtlpMeterRegistry.java:109)
	at io.micrometer@1.9.3//io.micrometer.core.instrument.push.PushMeterRegistry.publishSafely(PushMeterRegistry.java:54)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

Reply:
You're right. The code has changed since I wrote that. I'll update it.